daml/Tests/Distribution/Auction.daml

```diff
@@ -28,10 +28,10 @@ data Fixture = Fixture with
   bobDepositCid : ContractId AssetDeposit
   charlieDepositCid : ContractId AssetDeposit
   daveDepositCid : ContractId AssetDeposit
-  aliceBidRequestCid : ContractId Bidding.R
-  bobBidRequestCid : ContractId Bidding.R
-  charlieBidRequestCid : ContractId Bidding.R
-  daveBidRequestCid : ContractId Bidding.R
+  aliceAuctionCid : ContractId Bidding.A
+  bobAuctionCid : ContractId Bidding.A
+  charlieAuctionCid : ContractId Bidding.A
+  daveAuctionCid : ContractId Bidding.A
```

Comment:
I'd probably avoid this `type` synonym unless the code is self-contained in this file.